packages/create-cloudflare/templates/hello-world-durable-object/js/src/index.js

```diff
@@ -52,16 +52,17 @@ export default {
 	 * @returns {Promise<Response>} The response to be sent back to the client
 	 */
 	async fetch(request, env, ctx) {
-		// We will create a `DurableObjectId` using the pathname from the Worker request
-		// This id refers to a unique instance of our 'MyDurableObject' class above
-		let id = env.MY_DURABLE_OBJECT.idFromName(new URL(request.url).pathname);
+		// Create a `DurableObjectId` for an instance of the `MyDurableObject`
+		// class named "foo". Requests from all Workers to the instance named
+		// "foo" will go to a single globally unique Durable Object instance.
+		let id = env.MY_DURABLE_OBJECT.idFromName("foo");
```

Comment:
Hey @joshthoward what is the purpose of changing the id in this PR?

```suggestion
		const id = env.MY_DURABLE_OBJECT.idFromName(new URL(request.url).pathname);
```

Reply:
@vicb it's not illustrative of the example.